server/src/main/java/com/example/tyfserver/donation/dto/DonationMessageRequest.java

```diff
@@ -1,15 +1,25 @@
 package com.example.tyfserver.donation.dto;
 
+import lombok.AccessLevel;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 
 @Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
 public class DonationMessageRequest {
 
-    private final String name;
-    private final String message;
+    private String name;
+    private String message;
+    private boolean isPublic;
 
     public DonationMessageRequest(final String name, final String message) {
         this.name = name;
         this.message = message;
```

Comment:
```suggestion
        this(name, messsage, false);
```
위와 같이 둬서 디폴트 값이 `false`임을 명시하는게 좋을 듯,  추가적으로 주생성자, 부생성자 관계를 띄도록 변경하면 좋을 것 같아

Reply:
좋아요!